schemas/dictionary.json

```diff
@@ -534,7 +542,7 @@
       "title": "Skip Initial Space",
       "description": "Specifies the interpretation of whitespace immediately following a delimiter. If false, whitespace immediately after a delimiter should be treated as part of the subsequent field.",
       "type": "boolean",
-      "default": true,
+      "default": false,
```

Comment:
That's unrelated to this work, I believe this file was not built in https://github.com/frictionlessdata/specs/commit/26cca8e50fd5487581f96aec105f45461957c00d